### DISCUSSION TITLE
No html content on webpage

### DISCUSSION THREAD

**Discussion by wassa350** [arendst/Tasmota#18939] [Q&A]
I have a working Sonoff Mini running Tasmota that I flashed using DIY mode. My issue is I had stored this mini for sometime and when I wanted to use it again it appears to have lost the html for the webpage. I can navigate "blind" to any buttons like TOGGLE and it functions OK, just cant see the gui. this is what i see at botom of page;

[Sonoff-Tasmota 6.7.1 by Theo Arends](https://bit.ly/tasmota)

**Comment by barbudor** [comment]
Strange
But consideing it's running a quite old version and you don't have it in use currently, I wouldn't spent time a go straight for a serial reflash from scratch with latest release

**Reply by sfromis** [reply]
If you want to debug, you could try using the web browser debugging console, or viewing HTML source. You might have some sort of browser extension sabotaging the page in the name of security theater.

But yeah, I do agree with taking the opportunity to upgrade to current Tasmota.
